### DISCUSSION TITLE
Sample code

### DISCUSSION THREAD

**Discussion by Taqi-Techverx** [conductor-oss/csharp-sdk#38] [Q&A] [answered]
Is there any sample code for .net 6 to use this ?

**Comment by manan164** [comment]
Hi @Taqi-Techverx , We are in process of upgrading the SDK. I will update the thread with a sample for .net 6 once we do it. Thanks.

**Reply by Taqi-Techverx** [reply]
Thanks for update

**Comment by ghost** [comment, marked as answer]
Hey @Taqi-Techverx, we have made a few releases recently, with [0.1.6](https://github.com/conductor-sdk/conductor-csharp/releases/tag/0.1.6) being the latest one.

You can take a look at our brand new documentation. Feedbacks are appreciated!
- [Overall docs](https://github.com/conductor-sdk/conductor-csharp/blob/main/README.md)
- [Worker docs](https://github.com/conductor-sdk/conductor-csharp/blob/main/docs/readme/workers.md)
  - There is a known change from previous version, which is the hosted service setup, from `AddHostedService` to `WithHostedService`. We can work it out if it's a blocker :)
